js/packages/cli/src/helpers/upload/aws.ts

```diff
@@ -59,7 +60,8 @@ export async function awsUpload(
   });
   const updatedManifestBuffer = Buffer.from(JSON.stringify(manifestJson));
 
-  const metadataFilename = filename.replace(/.png$/, '.json');
+  const extensionRegex = new RegExp(`/${imageExt}$/`);
```

Comment:
By combining both regexp literal syntax (`/xyz/`) and the regex constructor `new RegExp()`, the resulting regexp does not match the file extension and breaks AWS uploads. 

Please change to the following:

```
const extensionRegex = new RegExp(`${imageExt}$`);
```